roles/openshift_metering/vars/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+openshift_metering_namespace: openshift-metering
```

Comment:
by no means a blocker, there is a pattern to prefix 'private' variables with "__" so this could be `__openshift_metering_namespace` which implies its private and shouldn't really be changed (but obviously you still can).

Reply:
That's a common pythonism but I grepped the codebase for this pattern and didn't find it in any yaml file.

Reply:
Hmm, I know we do it in the logging roles. It may not be very wide spread anymore. It was a common practice before with our image names

https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_logging_elasticsearch/vars/main.yml

Reply:
Heh, maybe I didn't give it enough time. I can do this.